packages/components/src/Components/InvalidObject/InvalidObject.js

```diff
@@ -10,9 +10,9 @@ import { withRouter } from 'react-router-dom';
 const InvalidObject = ({ history, ...props }) => {
     return (
         <section {...props} className="pf-l-page__main-section pf-c-page__main-section ins-c-component_invalid-componet">
-            <Title size='3xl'>This doesn&apos;t seem to exist.</Title>
+            <Title headingLevel="h2" size='3xl'>This doesn&apos;t seem to exist.</Title>
```

Comment:
This should be `headingLevel="h1"`

Reply:
Should it? Regarding the docs, the default value used to be `h2`: https://www.patternfly.org/v4/documentation/react/overview/upgrade-guide#title

Reply:
Check the snapshots it used to be `h1`, looks like the docs are lying...

Reply:
Well the docs says `h2`, but the code says `h1`. Nice to know that I can't relly on the upgrade guide.

https://github.com/patternfly/patternfly-react/pull/3922/files#diff-29db5c83c2f983f2d91ed0f5ffaf2817L31